### PR TITLE
Adding Titan Takeoff Integration and TitanML logo

### PR DESCRIPTION
**Pull request by rorcde**
Including example on how to create a daily news digest with TitanML Takeoff and Deepset Haystack

### CODE REVIEW

File: integrations/titanml-takeoff.md [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+        github: fergusbarratt
+        twitter: BarrattFergus
+        linkedin: https://www.linkedin.com/in/fergusfinn/
+    - name: Rod Rivera
+      socials:
+        github: rorcde
+        twitter: rorcde
+        linkedin: https://www.linkedin.com/in/aiengineer/
+pypi: https://pypi.org/project/takeoff-haystack/
+repo: https://github.com/deepset-ai/haystack-core-integrations/tree/main/integrations/titanml-takeoff
```
Comment (TuanaCelik) [inline integrations/titanml-takeoff.md:17]: This should be the repo of your integration as we don't host the code here

File: integrations/titanml-takeoff.md [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+type: Model Provider
+report_issue: https://github.com/deepset-ai/haystack-core-integrations/issues
```
Comment (TuanaCelik) [inline integrations/titanml-takeoff.md:19]: same here, this should be your issues

File: integrations/titanml-takeoff.md [hunk 3]
```diff
@@ -0,0 +1,130 @@
...
+
+```bash
+docker run --gpus all -e TAKEOFF_MODEL_NAME=TheBloke/Llama-2-7B-Chat-AWQ \
+                      -e TAKEOFF_DEVICE=cuda \
+                      -e TAKEOFF_MAX_SEQUENCE_LENGTH=256 \
+                      -it \
+                      -p 3000:3000 tytn/takeoff-pro:0.11.0-gpu
+```
+
+## TextGeneration
```
Comment (TuanaCelik) [inline integrations/titanml-takeoff.md:54]:
```suggestion
## Text Generation
```

**Comment by rorcde** [comment]
In the latest commit, we updated the URLs and also made changes to the text. Feedback, @TuanaCelik?

**Review by TuanaCelik — APPROVED** [review]

**Pull request merged**